up42/job.py

```diff
@@ -42,41 +43,51 @@ def __init__(
         self.results = None
         self.order_ids = order_ids
         if self.auth.get_info:
-            self.info = self._get_info()
+            self._info = self.info
 
     def __repr__(self):
         return (
             f"Job(job_id={self.job_id}, project_id={self.project_id}, "
-            f"order_ids={self.order_ids}, auth={self.auth}, info={self.info})"
+            f"order_ids={self.order_ids}, auth={self.auth}, info={self._info})"
         )
 
-    def _get_info(self):
-        """Gets metadata info from an existing Job"""
+    @property
+    def info(self) -> Dict:
+        """
+        Gets the job metadata information.
+        """
         url = f"{self.auth._endpoint()}/projects/{self.project_id}/jobs/{self.job_id}"
         response_json = self.auth._request(request_type="GET", url=url)
-        self.info = response_json["data"]
-        return self.info
+        self._info = response_json["data"]
+        return self._info
 
-    def get_status(self) -> str:
+    @property
+    def status(self) -> str:
```

Comment:
Why the renaming? In case of the SDK, renaming always leads to breaking code which uses this functionality.

Reply:
Good question, feels very "natural" to use to have these as object properties, even though does a request under the hood. SDK requires an internet connection anyway. Rodrigo added some status queries as class properties and we really liked it. Initial motivation was to consolodiate the naming with the differently called jobcollection functions, now combined with the property change.

Reply:
Merci